integration/tests/test_utils.py

```diff
@@ -229,18 +233,16 @@ def success_predicate(result):
     return spin(fn, success_predicate).json()
 
 
-def _nested_dict_merge(a, b, path=None):
-    "ripped from http://stackoverflow.com/questions/7204805/dictionaries-of-dictionaries-merge"
-    if path is None: path = []
-    a = a.copy()
-    for key in b:
-        if key in a:
-            if isinstance(a[key], dict) and isinstance(b[key], dict):
-                _nested_dict_merge(a[key], b[key], path + [str(key)])
-            elif a[key] == b[key]:
-                pass # same leaf value
-            else:
-                raise Exception('Conflict at %s' % '.'.join(path + [str(key)]))
+def _merge_dictionary(dict1, dict2):
+    if (not isinstance(dict2, dict)):
+        return dict1
+    ret = {}
+    for k, v in dict1.items():
+        ret[k] = v
```

Comment:
Am I missing why this isn't
```
ret = dict1.copy()
```
Maybe I'm guilty of playing code golf here, but if they're the same it seems more obvious.